### PR TITLE
feat: add config to disable bridge tokens in the ui

### PR DESCRIPTION
**Pull request by dohaki**
Closes ACX-1100

Allows us to enable USDT in the API but disable temporarily in the UI

### CODE REVIEW

File: src/views/Bridge/hooks/useBridge.ts [hunk 1]
```diff
@@ -29,7 +29,7 @@ import { BridgeLimitInterface } from "utils/serverless-api/types";
 import { useBridgeAction } from "./useBridgeAction";
 import { useBridgeDepositTracking } from "./useBridgeDepositTracking";
 
-const enabledRoutes = getConfig().getRoutes();
+const enabledRoutes = getConfig().getEnabledRoutes();
```
Comment (james-a-morris) [inline src/views/Bridge/hooks/useBridge.ts:32]: Does this also apply to any pools?

**Review by james-a-morris — APPROVED** [review]
Left a comment. Feel free to disregard if it isn't actionable

File: src/views/Bridge/hooks/useBridge.ts (hunk 1 above)
Comment (dohaki) [reply src/views/Bridge/hooks/useBridge.ts:32]: Hmm, good point. Let me check with @nicholaspai

Comment (dohaki) [reply src/views/Bridge/hooks/useBridge.ts:32]: So the config should only apply for bridging

**Review by nicholaspai — APPROVED** [review]

**Pull request merged**